### PR TITLE
Creating installation file

### PR DESCRIPTION
**Pull request by omarcopires**

### CODE REVIEW

File: classes/database_mysql.php [hunk 1]
```diff
@@ -26,15 +26,15 @@ public function connect()
 
 	public function fieldName($name)
 	{
-		if(strspn($name, "1234567890qwertyuiopasdfghjklzxcvbnm_") != strlen($name))
+		if(strspn($name, "1234567890_aAaQqWwEeRrTtYyUuIiOoPpAaSsDdFfGgHhJjKkLlZzXxCcVvBbNnMm_") != strlen($name))
```
Comment (jlcvp) [inline classes/database_mysql.php:29]: I recommend using regex and preg_match for cleaner code

this if would be:
```php
if(preg_match("/[^a-zA-Z0-9_]+/", $name) > 0)
```

File: install.php [hunk 2]
```diff
@@ -0,0 +1,567 @@
...
+		$columns[] = array('players', 'hide_set', 'INT', '11', NULL);
+		$columns[] = array('players', 'hide_skills', 'INT', '11', NULL);
+		$columns[] = array('players', 'marriage_spouse', 'INT', '11', NULL);
+		$columns[] = array('players', 'marriage_status', 'BIGINT', '20', '0');
+		$columns[] = array('players', 'name_of_column', 'TINYINT', '1', '0');
+		$columns[] = array('players', 'signature', 'VARCHAR', '255', '');
+
+		$tables = array();
+		// mysql tables
+		$tables[Database::DB_MYSQL]['announcements'] = "CREATE TABLE `announcements` (
```
Comment (jlcvp) [inline install.php:243]: What happens if those tables are already created?

Comment (omarcopires) [reply install.php:243]: > What happens if those tables are already created?

I applied the change to create only if it doesn't exist.

File: classes/database_mysql.php (hunk 1 above)
Comment (omarcopires) [reply classes/database_mysql.php]: > I recommend using regex and preg_match for cleaner code
> 
> this if would be:
> 
> ```
> if(preg_match("/[^a-zA-Z0-9_]+/", $name) > 0)
> ```

I applied the suggested changes

**Pull request merged**